src/test/ui/issues/issue-65673.stderr

```diff
@@ -1,13 +1,13 @@
 error[E0277]: the size for values of type `(dyn Trait + 'static)` cannot be known at compilation time
-  --> $DIR/issue-65673.rs:9:5
+  --> $DIR/issue-65673.rs:9:16
    |
+LL | trait WithType {
+   |       -------- required by a bound in this trait
 LL |     type Ctx;
-   |          --- associated type defined here
+   |     --------- required by this bound in `WithType`
```

Comment:
I wonder if we want to be careful about the implied `Sized` bounds? Perhaps something we can leave for follow-up.